### PR TITLE
Release v0.77.0

### LINKED ISSUE #1726: Release the operator v0.77.0

### PR DESCRIPTION
**Pull request by yuriolisa**
Resolves #1726

### CODE REVIEW

File: versions.txt
Comment (pavolloffay) [inline versions.txt:1]: java version needs to be updated.

**Review by pavolloffay — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]

**Comment by jaronoff97** [comment]
@TylerHelmuth i merged #1740 so when Yuri rebases it'll be included here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@ Changes by Version
 ==================
 
 <!-- next version -->
+## 0.77.1
```
Comment (TylerHelmuth) [inline CHANGELOG.md:5]:
```suggestion
## 0.77.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@ Changes by Version
...
+
+### 💡 Enhancements 💡
```
Comment (TylerHelmuth) [inline CHANGELOG.md:7]: We're missing the `Add support for Go auto instrumentation` changelog entry.

**Review by TylerHelmuth — APPROVED** [review]

**Comment by jaronoff97** [comment]
@TylerHelmuth gonna merge this when the tests pass. I can handle the version tagging... i wanna discuss if we want to automate that process at the next SIG meeting btw.

**Pull request merged**